src/main/kotlin/uk/gov/justice/digital/hmpps/makerecalldecisionapi/controller/CaseSummaryController.kt

```diff
@@ -0,0 +1,31 @@
+package uk.gov.justice.digital.hmpps.makerecalldecisionapi.controller
+
+import io.swagger.v3.oas.annotations.Operation
+import org.slf4j.LoggerFactory
+import org.springframework.http.MediaType
+import org.springframework.security.access.prepost.PreAuthorize
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RequestParam
+import org.springframework.web.bind.annotation.RestController
+import uk.gov.justice.digital.hmpps.makerecalldecisionapi.domain.SearchByCrnResponse
+import uk.gov.justice.digital.hmpps.makerecalldecisionapi.service.CaseSummaryService
+
+@RestController
+@RequestMapping(produces = [MediaType.APPLICATION_JSON_VALUE])
+class CaseSummaryController(
```

Comment:
Can we rename this controller to something more related to 'searching'?

Reply:
done :)